### PR TITLE
AWS v16.5.2

### PR DESCRIPTION
**Pull request by njuettner**
Issue: https://github.com/giantswarm/roadmap/issues/1632

### CODE REVIEW

**Comment by njuettner** [comment]
We need to wait for the aws-operator for v16.5.2, this PR https://github.com/giantswarm/aws-operator/pull/3439 needs to get merged. 
But it's already prepared to run tests

File: aws/v16.5.2/README.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# :zap: Giant Swarm Release v16.5.2 for AWS :zap:
+
+This is a patch release to add missing Elastic File System (EFS) IAM permissions. It also allows EFS CSI Driver to get the Instance Identity from the Instance Metadata Service (IMDS).
+
+## Change details
+
+
+### aws-operator [10.14.2](https://github.com/giantswarm/aws-operator/releases/tag/v10.14.2)
```
Comment (njuettner) [inline aws/v16.5.2/README.md:8]: this will be the new version of aws-operator

**Review by whites11 — APPROVED** [review]

**Review by T-Kukawka — APPROVED** [review]

**Pull request merged**